app/models/concerns/non_passported_state_machine.rb

```diff
@@ -6,6 +6,7 @@ class NonPassportedStateMachine < BaseStateMachine
     state :checking_citizen_answers
     state :analysing_bank_transactions
     state :provider_assessing_means
+    state :assessing_partner_means
```

Comment:
just on naming, and looking at the state above it, would `provider_assessing_partner_means` be clearer. but no biggee

Reply:
assessing_client_means would be a better name but that is out of scope of this ticket! 🙂